paper.tex

```diff
@@ -242,7 +242,10 @@ \subsubsection{Model Description}
 
 Al-Fagih et al. presented\cite{DataPrice} $P_s$ is a sigmoid function of $P_r$, and based on rule of thumb $C_{maintain}$ is about a exponential function of $P_r$. We can substitute $P_s$ and $C_{maintain}$ with $P_r$ into Eq. (\ref{payoff}), then we can find out the Nash Equilibrium of repeated game easily.
 
-\section{\normalsize\textbf{Conclusion}}
+\section{\normalsize\textbf{Future Work}}
+%Data Auction
+The data auction process is one of the most popular data trading mechanisms and an economically-diven scheme that establishes corresponding prices of commodities through bidding process between consumers' and providers'. While there has been many auction models\cite{BigPicDataMarket} in several areas, however, only a few of research focus on third-party auction platform. And there're a lot of challeneges such as, protecting both consumers' and providers' privacy, recording auction process and analyzing potential threats.
```

Comment:
You did not address the challenges which are directly related to this research.

Reply:
@jserv ,
I fixed some typos also, please review again.